service.go

```diff
@@ -159,6 +172,23 @@ func (s *serviceFactory) registeredServiceIDs() []ServiceID {
 	return ids
 }
 
+// generateKeyPairs generates the key pairs for the services that
```

Comment:
It is surprising to me that key generation is not in the appcfg directory. It seems like something that ONLY happens during config file creation, not automagically here in onet. (But if you can explain your reasoning, then I wouldn't be surprised and would be ok with this.)